dist/api_data.py

```diff
@@ -145,6 +157,18 @@ def __ge__(self, other):
     Config('disaggregated', '', r'''
         configure disaggregated storage for this file''',
         type='category', subconfig=disaggregated_config_common + [
+            Config('delta_pct', '20', r'''
```

Comment:
It seems weird that these are in the disaggregated specific section - is there a ticket to review the configuration options already?

Reply:
Yep, https://jira.mongodb.org/browse/WT-14721.